.storybook/config.js

```diff
@@ -1,6 +1,9 @@
 import { configure } from '@storybook/react'
+import { setAddon } from '@storybook/react'
 import { setOptions } from '@storybook/addon-options'
 import 'react-chromatic/storybook-addon'
+import JSXAddon from 'storybook-addon-jsx'
+setAddon(JSXAddon)
```

Comment:
Are storybook changes/additions needed for the TypeScript migration?

Reply:
It's no must, if it's becoming a bottleneck i can leave it out but if it's a succesful port to TSX I think it's better to make as much as possible Typescript :D